waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/clients/WavesBlockchainGrpcAsyncClient.scala

```diff
@@ -77,57 +76,57 @@ class WavesBlockchainGrpcAsyncClient(channel: ManagedChannel, monixScheduler: Sc
     case _: Result.Denied             => RunScriptResult.Denied
   }
 
-  /** Performs new gRPC call for receiving of the spendable balance changes stream */
-  def requestBalanceChanges(): Unit = balancesService.getBalanceChanges(Empty(), balanceChangesObserver)
-
   /** Returns stream of the balance changes as a sequence of batches */
-  def spendableBalanceChanges: Observable[SpendableBalanceChanges] = spendableBalanceChangesSubject
+  override lazy val spendableBalanceChanges: Observable[SpendableBalanceChanges] = {
+    requestBalanceChanges()
+    spendableBalanceChangesSubject
+  }
 
-  def spendableBalance(address: Address, asset: Asset): Future[Long] = handlingErrors {
+  override def spendableBalance(address: Address, asset: Asset): Future[Long] = handlingErrors {
```

Comment:
Why do you use `override` here? It's optional

Reply:
IDEA adds this modificator when I use "implement methods of interface" :) 
Also I always use `override` because If a method in a base class was removed, I want to remove this method from descendants too.

Reply:
Also it easier to find an issue, when arguments are changed in a base trait method